### PR TITLE
tuntap deleted

### PR DESCRIPTION
**Pull request by AnastasiaSulyagina**
See [#15603](https://github.com/caskroom/homebrew-cask/issues/15603) for more details

### CODE REVIEW

**Comment by ilovezfs** [comment]
This would be more palatable if the Casks supported source builds where possible as well. It seems like going back to the Dark Ages to take "formula knowledge" of how to build things and just toss it into the boneyard because some pkg/dmg is "preferred."

**Comment by MikeMcQuaid** [comment]
@ilovezfs Given we're relying on an upstream signed kernel extension in this case: it doesn't really matter if we have a formula as it can't be used anyway. Regardless, I agree if upstream do not provide binaries and disagree if they do.

**Comment by ilovezfs** [comment]
That's like saying because a stable tag exists, a head spec should not be provided. I don't see how one follows from the other.

**Comment by MikeMcQuaid** [comment]
@ilovezfs Well, in this case you cannot use the formula on 10.10 and above and we've killed many others for far less. As to the other point: Homebrew was never intended for GUI applications and we handle them far worse than Cask does.

**Comment by ilovezfs** [comment]
I guess since I have kext signing certificates my perspective is completely warped beyond repair at this point. Oh well.

**Comment by DomT4** [comment]
Cross-posting myself since the comment applies here too: https://github.com/Homebrew/homebrew-core/pull/2076#issuecomment-226948639

**Comment by MikeMcQuaid** [comment]
@Homebrew/maintainers Any final (very strong) objections before I merge this?

**Comment by DomT4** [comment]
We need to fix the Requirements first. That's all from me.

**Comment by MikeMcQuaid** [comment]
Cool, thanks @DomT4. @AnastasiaSulyagina we'll talk more about that later.

**Comment by MikeMcQuaid** [comment]
Requirements fix in https://github.com/Homebrew/brew/pull/487.

File: tap_migrations.json [hunk 1]
```diff
@@ -279,5 +279,6 @@
   "xspringies": "homebrew/x11",
   "xulrunner": "homebrew/boneyard",
   "yarp": "homebrew/x11",
-  "ydict": "homebrew/boneyard"
+  "ydict": "homebrew/boneyard",
+  "tuntap": "caskroom/cask"
```
Comment (MikeMcQuaid) [inline tap_migrations.json]: Can you make sure this is in alphabetical order? Thanks!

**Comment by MikeMcQuaid** [comment]
@AnastasiaSulyagina 😻

**Pull request merged**